fuel/schemes.py

```diff
@@ -232,3 +232,17 @@ def get_request_iterator(self):
         indices = list(self.indices)
         self.rng.shuffle(indices)
         return iter_(indices)
+
+@add_metaclass(ABCMeta)
+class LimitedScheme(IterationScheme):
+    """Iterate over the sequence given an iterationScheme object
+        but in a limited number of excursions defined by times
+    """
+    def __init__(self, iteration_scheme, times, *args, **kwargs):
```

Comment:
You can safely remove `*args, **kwargs`. It's used in other iterators (e.g. `ShuffledExampleScheme`) to pass arbitrary args and kwargs to the superclass' `__init__` method, but in this case you're not calling any other `__init__` method.